docs/cloud-tutorials/application-management.md

```diff
@@ -66,18 +77,35 @@ npx dbos-cloud app versions <app-name>
 
 You can redeploy a previous version of your application by passing `--previous-version <version-id>` to the [`app deploy`](../api-reference/cloud-cli.md#npx-dbos-cloud-app-deploy) command.
 
-```
+```shell
 npx dbos-cloud app deploy --previous-version <version-id>
 ```
 
 However, this will fail if the previous and current versions have different database schemas.
+
+Also, note DBOS Cloud will keep running Transact workflows from previous versions until they complete. This means you need to be mindful of non-backward compatible database schema migrations, as they could break running workflows.
```

Comment:
Do we have any safeguards/warnings in place for that? Or would it just break a running workflow from an old version if I deploy a non-backward compatible schema migration while an old workflow is still running?

Reply:
No safeguards at the moment. To be fair, this problem isn't unique to Transact--if you do rolling deployments of an app to any service, you'll hit the same problem with non-backwards-compatible migrations (which is why many shops won't do such migrations at all, they're too risky).